### PR TITLE
Feature/image article

### PR DESCRIPTION
**Pull request by KarthikTG47**

### CODE REVIEW

File: bootcamp-demo-project/core/src/main/java/com/bootcamp/core/models/Article.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+package com.bootcamp.core.models;
```
Comment (RaghavendraNagaraj) [inline bootcamp-demo-project/core/src/main/java/com/bootcamp/core/models/Article.java]: Create a new package for this and rename your class

File: bootcamp-demo-project/ui.apps/src/main/content/jcr_root/apps/bootcamp-demo-project/components/structure/articles/content.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<div data-sly-resource="${ @path='contentpar', resourceType='foundation/components/parsys'}"></div>
```
Comment (RaghavendraNagaraj) [inline bootcamp-demo-project/ui.apps/src/main/content/jcr_root/apps/bootcamp-demo-project/components/structure/articles/content.html]: What is this and why is this required?

File: bootcamp-demo-project/core/src/main/java/com/bootcamp/core/models/Article.java (hunk 1 above)
Comment (KarthikTG47) [reply bootcamp-demo-project/core/src/main/java/com/bootcamp/core/models/Article.java]: Okay, sir. Will do it.

File: bootcamp-demo-project/ui.apps/src/main/content/jcr_root/apps/bootcamp-demo-project/components/structure/articles/content.html (hunk 2 above)
Comment (KarthikTG47) [reply bootcamp-demo-project/ui.apps/src/main/content/jcr_root/apps/bootcamp-demo-project/components/structure/articles/content.html]: This has been removed as discussed, Sir.
